scene/gui/tree.h

```diff
@@ -239,7 +239,7 @@ class TreeItem : public Object {
 	TreeItem *get_children();
 
 	TreeItem *get_prev_visible();
-	TreeItem *get_next_visible();
+	TreeItem *get_next_visible(bool p_wrap = true);
```

Comment:
As discussed on IRC, this should default to `false` to preserve compatibility, and it's also more intuitive not to wrap around by default IMO.
I would add the same parameter to `get_prev_visible` for consistency, and `"wrap"` needs to be added in the `bind_method` call, otherwise the docs will know this argument as "arg".

Reply:
Finally got to fix it. The reason `get_prev_visible()` couldn't wrap previously was that I didn't really know how it could work (also I thought it's an internal method, not exposed to GDScript). Well, right now it does a while loop over all elements, but not sure if it couldn't be done better.

Reply:
Ah right, I didn't expect that `get_prev_visible(wrap)` would need a while loop to find that last visible item. That's a bit expensive, but since it's not the default behaviour, I guess that's fine.